src/scrabble.js

```diff
@@ -1,5 +1,48 @@
-function scrabble() {
-  // write code here
+const scrabbleValue = {
```

Comment:
I like the layout of the list since it's easy on the eye's and very easy to understand